### PR TITLE
adopt EDB licensing multiarch image

### PR DESCRIPTION
**Pull request by Daniel-Fan**

### CODE REVIEW

File: controllers/constant/odlm.go [hunk 1]
```diff
@@ -117,15 +117,6 @@ spec:
               spec:
                 imagePullSecrets:
                   - name: ibm-entitlement-key
-                affinity:
```
Comment (horis233) [inline controllers/constant/odlm.go:120]:
```
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: kubernetes.io/arch
                operator: In
                values:
                - amd64
                - ppc64le
                - s390x
```

Could we update node affinity to this?

Comment (horis233) [reply controllers/constant/odlm.go:120]: @Daniel-Fan ^^^

Comment (Daniel-Fan) [reply controllers/constant/odlm.go]: @horis233 Thank you for the reminder!

Updated~

**Review by horis233 — APPROVED** [review]
/lgtm

File: controllers/constant/odlm.go [hunk 2]
```diff
@@ -140,7 +142,7 @@ spec:
                     data:
                       EDB_LICENSE_KEY: $(base64 /license_keys/edb/EDB_LICENSE_KEY | tr -d '\n')
                     EOF
-                  image: cp.icr.io/cp/cpd/wd-postgres-license@sha256:4d8d0ecd31d04e15f757ffda74101ef5bd7fb5789db1ffcef892e961ef312ebf
+                  image: icr.io/cp/cpd/edb-postgres-license-provider@sha256:607231e7337608278220853bd0bb04002488a6e74d4418752b1720133daef036
```
Comment (horis233) [inline controllers/constant/odlm.go:145]: @Daniel-Fan We need to change the registry to `cp.icr.io`

Comment (Daniel-Fan) [reply controllers/constant/odlm.go]: Thank you for the reminder! Updated~

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**